selene-lib/src/lints.rs

```diff
@@ -37,6 +37,9 @@ pub mod unused_variable;
 #[cfg(feature = "roblox")]
 pub mod roblox_incorrect_color3_new_bounds;
 
+#[cfg(feature = "roblox")]
+pub mod roblox_mismatched_udim2_new_arg_count;
```

Comment:
Nit: This should be `roblox_suspicious_udim2_new`, because it's not technically incorrect.